1-mandatory/3-slideshow/index.html

```diff
@@ -11,7 +11,17 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <!-- Write your HTML in here -->
+    <div id="main">
+      <img src="https://images.unsplash.com/photo-1475924156734-496f6cac6ec1?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTd8fG5hdHVyZXxlbnwwfHwwfHw%3D&auto=format&fit=crop&w=800&q=60">
```

Comment:
Remember to add the id for the testing to work. For example, in the tests you can see that it's looking for the id #carousel-image: https://github.com/CodeYourFuture/JavaScript-Core-2-Coursework-Week4/blob/main/1-mandatory/3-slideshow/slideshow.test.js#L43

Reply:
yes you are right.thanks